addons/website_slides/static/src/scss/slide_views.scss

```diff
@@ -11,6 +11,11 @@
     }
 }
 
+.o_inner_image {
```

Comment:
this classname is generic (not prefixed by any module name), it looks like a framework classname => rename it to o_website_slides_inner_image

Also, the selector is too weak. As soon as this addon is installed, any element in any addon with classname `o_inner_image` will come with those css rules. You want to scope these rule to the kanban view where it is used.